### PR TITLE
Curated carousel

### PR DESCRIPTION
**Pull request by stacysealky**
Curated carousel displays when there is 0 wishcard to be donated, so that the home page looks better.

### CODE REVIEW

**Comment by stacysealky** [comment]
It's throwing error because of the `curated.push(card);` @Enubia

**Comment by Enubia** [comment]
> It's throwing error because of the `curated.push(card);` @Enubia 

Yes, because your are missing a proper type

File: src/db/repository/WishCardRepository.ts [hunk 1]
```diff
@@ -50,6 +49,32 @@ export default class WishCardRepository {
...
+			'5f7f8a8eb09ed5121216bd1b',
+			'5fc48548495483e2e4e87432',
+			'5f90a7fc987faa6593d8c061',
+			'5fd18775e7553d5923e6ce63',
+			'5fd188fee7553d5923e6ce64',
+			'5f9b3373987faa6593d8c07a',
+			'5fc486a5495483e2e4e87433',
+			'5f847114a99508e349bb87fd',
+		];
+		const curated = [];
```
Comment (Enubia) [inline src/db/repository/WishCardRepository.ts:68]:
```suggestion
		const curated: WishCard[] = [];
```

Comment (Enubia) [reply src/db/repository/WishCardRepository.ts:68]: this should fix the build error

**Review by Enubia — CHANGES_REQUESTED** [review]

**Review by Enubia — APPROVED** [review]

**Pull request merged**